chapters/config.Rmd

```diff
@@ -319,9 +321,11 @@ to pass a YAML file full of Matplotlib parameters to `plotcounts.py`.\index{conf
 First,
 we save the parameters we want to change in a file inside our project directory.
 We can call it anything,
-but `plotparams.yml` seems like it will be easy to remember:
+but `plotparams.yml` seems like it will be easy to remember,
```

Comment:
end the sentence after 'remember' and get rid of the 'and' on the following line?